test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -74,6 +74,8 @@
                 "44716",
                 "44717",
                 "43453",
+                # tensorflow-estimator and tensorflow versions must match. For all TF versions below TF 2.9.0, we cannot upgrade tf-estimator to 2.9.0
+                "48551",
```

Comment:
Does this affect inference images? And why is it only on the allowlist for inference, and not for training?